pkg/log/log.go

```diff
@@ -10,7 +10,7 @@ const (
 	ColorGreen  = "\033[32m"
 	ColorBlue   = "\033[34m"
 	ColorYellow = "\033[33m"
-	ColorWhite  = "\033[37m"
+	ColorWhite  = "\033[97m"
```

Comment:
This is the color control code for "bright white" which looks an awful lot like default white tbh, either way: it's brighter and easier to read than the gray.